src/emulator/downloadableEmulators.ts

```diff
@@ -480,7 +494,7 @@ async function _runBinary(
  * @param emulator
  */
 export function getDownloadDetails(emulator: DownloadableEmulators): EmulatorDownloadDetails {
-  const details = DownloadDetails[emulator];
+  const details = generateDownloadDetails()[emulator];
```

Comment:
I think we may as well make `generateDownloadDetails` take `emulator` as an argument